pandas/tests/indexes/multi/test_conversion.py

```diff
@@ -87,46 +85,6 @@ def test_to_hierarchical():
     assert result.names == index.names
 
 
-@pytest.mark.skipif(PY3, reason="testing legacy pickles not support on py3")
-def test_legacy_pickle(datapath):
-
-    path = datapath('indexes', 'multi', 'data', 'multiindex_v1.pickle')
-    obj = pd.read_pickle(path)
```

Comment:
can u also remove the legacy code itself from MultiIndex

Reply:
So, I came to the conclusion that I can just delete the `__setstate__` method as there is no associated `__getstate__` method.  I hope this approach is correct.